frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -93,6 +95,10 @@ export const eventUsageLogic = kea<
             extraProps?: Record<string, string | boolean | number | undefined>
         ) => ({ module, item, extraProps }),
         reportProjectHomeSeen: (teamHasData: boolean) => ({ teamHasData }),
+        reportEventSearched: (search_term: string, extra_props?: Record<string, number>) => ({
```

Comment:
nitpick but should these be camelcase instead? If not, when do we differentiate between snake or camel for these logic files?